src/Object/Api/Mastodon/Status.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * @copyright Copyright (C) 2020, Friendica
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Object\Api\Mastodon;
+
+use Friendica\BaseEntity;
+use Friendica\Util\DateTimeFormat;
+
+/**
+ * Class Status
+ *
+ * @see https://docs.joinmastodon.org/entities/status
+ */
+class Status extends BaseEntity
+{
+	/** @var string */
+	protected $id;
+	/** @var string (Datetime) */
+    protected $created_at;
```

Comment:
Code standards: Please use tabs for indentation.

Reply:
Done

Reply:
Please check the whole file, I didn't add a comment everywhere spaces were used.

Reply:
Done